rn/Teacher/ios/Teacher/Routes.swift

```diff
@@ -291,6 +289,15 @@ private func fileEditor(url: URLComponents, params: [String: String], userInfo:
     return CoreHostingController(FileEditorView(context: Context(path: url.path), fileID: fileID))
 }
 
+private func syllabus(url: URLComponents, params: [String: String], userInfo: [String: Any]?) -> UIViewController? {
+    guard ExperimentalFeature.nativeTeacherSyllabus.isEnabled else {
+        Router.open(url: url)
+        return nil
+    }
+    guard let courseID = params["courseID"] else { return nil }
+    return SyllabusViewController.create(courseID: ID.expandTildeID(courseID))
```

Comment:
nice remembering to expand IDs with tildes.

Reply:
the power of copypaste ;)